docs/hub/dduf.md

```diff
@@ -0,0 +1,166 @@
+     <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/DDUF/DDUF-Banner-dark.svg"/>
+</div>
+
+
+## Overview
+
+DDUF (**D**DUF’s **D**iffusion **U**nified **F**ormat) is a file format designed to make storing, distributing, and using diffusion models much easier. Built on the ZIP file format, DDUF offers a standardized, efficient, and flexible way to package all parts of a diffusion model into a single, easy-to-manage file.
+
+This work draws inspiration from the [GGUF](https://github.com/ggerganov/ggml/blob/master/docs/gguf.md) format.
+
+We've seeded some DDUFs of popular formats for the community to play with: https://huggingface.co/DDUF, check them out!
+
+<Tip>
+
+We welcome contributions with open arms!
+
+To create a widely adopted file format, we need early feedback from the community. Nothing is set in stone, and we value everyone's input. Is your use case not covered? Please let us know! Discussions about the DDUF format happen in the [DDUF organization](https://huggingface.co/spaces/DDUF/README/discussions/2).
+
+</Tip>
+
+## Motivation
+
+Yet, another file format? Yes, but for good reasons!
+
+The primary goal of DDUF is to create a community-endorsed single-file format for diffusion models. Current model distribution methods often involve multiple separate files, different weight-saving formats, and managing files from various locations. DDUF aims to solve these challenges by packaging all model components into a single file, enforcing a consistent structure while being opinionated about saving formats.
+
+The DDUF format is also designed to be language-agnostic. While we currently provide tooling for the Python ecosystem, there's nothing stopping similar tools from being developed in JavaScript, Rust, C++, and other languages. Like GGUF or safetensors, DDUF is built to be parsable from a remote location without downloading the entire file, which will enable advanced support on the Hugging Face Hub.
+
+## Key Features
+
+1. **Single file** packaging.
+2. Based on **ZIP file format** to leverage existing tooling.
+3. No compression, ensuring **`mmap` compatibility** for fast loading and saving.
+4. **HTTP-friendly**: metadata and file structure can be fetched remotely using HTTP Range requests.
+5. **Flexible**: each model component is stored in its own directory, following the current `diffusers` structure.
+6. **Safe**: uses `safetensors` as weights saving format and prohibits nested directories to prevent ZIP-bombs.
+
+## Technical specifications
+
+Technically, a `.dduf` file **is** a [`.zip` archive](https://en.wikipedia.org/wiki/ZIP_(file_format)). By building on a universally supported file format, we ensure robust tooling already exists. However, some constraints are enforced to meet diffusion models' requirements:
+- Data must be stored uncompressed (flag `0`), allowing mmap-compatibility.
```

Comment:
Would it make sense to have a one-liner about why map is needed here?

Reply:
I rephrased it to mention lazy-loading